template/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "dev": "node build/dev-server.js",
     "build": "node build/build.js"{{#unit}},
-    "unit": "karma start test/unit/karma.conf.js --single-run"{{/unit}}{{#e2e}},
+    "unit": "cross-env NODE_ENV=test karma start test/unit/karma.conf.js --single-run"{{/unit}}{{#e2e}},
```

Comment:
Probably best to use BABEL_ENV

Babel will use either of these, but will probably be more clear of the use and cause less side affects